content/articles/introduction-to-docker-swarm-in-container-orchestration/index.md

```diff
@@ -0,0 +1,65 @@
+
+It can also be termed as a Docker tool used for scheduling and network clustering. Docker Swarms enable you to manage containers when their size increases. 
+
+### Features of Docker Swarm
+The following are the main features of Docker Swarm:
+- **Security:** The nodes in Docker Swarm allow the enforcement of encryption and mutual authentication to enhance high security in communications between nodes. 
+- **Scaling:** Docker Swarm consists of automatic addition or removal of tasks to allow users to scale up or down as per their needs. 
+- **Decentralized design:** It consists of a decentralized design that allows you to create a swarm from one disk image. 
+- **Integration:** The cluster management of Docker Swarm has been integrated with Docker Engine. This allows users to manage swarms without requiring another orchestrations software. 
+- **Rolling updates:** During rollout, service updates on nodes can be made incrementally. In case of a problem, you can roll back to a previous safe service.
+- **Declarative service:** Docker Swarm consists of a declarative service model that allows users to define the required state of services in the Swarm environment. 
+- **Service discovery:** The manager nodes assign a DNS to each service. This means you can use an embedded DNS server to query a container that runs within the swarm.
+
+### How developers can use Docker Swarm
+Docker Swarms are important to developers, especially in the Docker ecosystem. They enable them to create and deploy Docker node clusters. Developers can also use Docker Swarm to perform the following tasks:
+- To scale up or down the size of containers.
+- To enhance the coordination of two or more containers. 
+- To allocate tasks to various container clusters. 
+- To roll out service updates on nodes.
+- To manage multiple container clusters, including their lifecycle and functionality. 
+- To reschedule containers in case of node failure.
+
+### How Docker Swarm Works
+Docker Swarm uses services, tasks, and nodes to manage containers. Let’s gain an overview of these aspects before diving into how they work. 
+
+A service is a list of tasks that need to be executed on nodes. Services allows app images to be deployed to the Docker Swarm. They enable users to launch containers. Examples of services include HTTP Servers, DB Servers, and Web Servers. 
+
+A task refers to work that needs to be done. A Swarm node is a singular Docker Engine or an instance of it. It consists of two types of nodes: worker nodes and manager nodes. 
+
+One or more nodes should be deployed before a service can be deployed. A service is created by setting up commands. The manager node consists of an API that connects to the Swarm environment. This enhances container orchestration through the creation of tasks for every service. These tasks are allocated and dispatched to a worker node. The worker node checks for tasks, receives tasks and executes them.

```

Comment:
add another paragraph to explain how docker swarm works. This will add value to the article